include/trace/events/sof_intel.h

```diff
@@ -27,7 +28,121 @@ TRACE_EVENT(sof_intel_hda_irq,
 		__assign_str(device_name, dev_name(sdev->dev));
 		__assign_str(source, source);
 	),
-	TP_printk("device_name=%s source=%s", __get_str(device_name), __get_str(source))
+	TP_printk("device_name=%s source=%s",
+		  __get_str(device_name), __get_str(source))
```

Comment:
This line splitting can be moved to the previous patch.

Reply:
The fix to add the 'space'  between `,` and `__get_str(source)` can be moved down.

Reply:
argh Git strikes again, fixed now.